css/details.css

```diff
@@ -0,0 +1,7 @@
+#details gaia-list {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  height: calc(100% - 50px);
+  overflow: scroll;
+}
```

Comment:
otherwise we scroll the whole panel (and the header with the back button gets out of the viewport).
But I forgot to do it for the main screen (the edit screen is not affected), I'll put the change for both in `app.css`.